src/runtime_src/core/tools/xbmgmt2/flash/xmc.cpp

```diff
@@ -28,7 +28,13 @@
 #include "core/common/utils.h"
 #include "core/common/system.h"
 #include "core/common/device.h"
+#include "core/common/error.h"
 #include "core/common/query_requests.h"
+#include "core/tools/common/XBUtilities.h"
+#include "core/tools/common/ProgressBar.h"
+namespace XBU = XBUtilities;
```

Comment:
This looks suspiciously like a circular dependency.  I expect xbmgmt2 -> xbmgmt2/flash , but not the other way around.